### DISCUSSION TITLE
Cursor becomes invisible on space chars at start of line.

### DISCUSSION THREAD

**Discussion by ChrisTaylorDeveloper** [projekt0n/github-nvim-theme#313] [General]
I'm using the github_light theme and it is the best light theme I've used so far.  Thank you.

On lines which begin with space characters, before text characters, the cursor turns invisible when sitting on-top of one of the space characters.

Is this by design or a potential problem with the theme?  If this is the intended behavior - how can I turn it off?  I would like to see the cursor at all times.

**Comment by ful1e5** [comment]
@ChrisTaylorDeveloper Thank you for your kind words!

Regarding the cursor issue, it's worth noting that the cursor color is inherited from your terminal. The colorscheme influences the cursor color, and this is particularly visible in neovim GUIs like [Neovide](https://github.com/neovide/neovide), [neovim-qt](https://github.com/equalsraf/neovim-qt), and others. However, when running Neovim inside a terminal, it inherits the cursor color from your terminal settings, no longer utilizing the color scheme colors.

If your cursor is becoming invisible on white space characters, it might be due to your terminal's dark color scheme conflicting with the light color scheme in Neovim. **To address this, try setting your terminal theme to light colors.** 

Additionally, you can download the GitHub color scheme from the [github-theme-contrib](https://github.com/projekt0n/github-theme-contrib) repository.

I hope this helps!

**Reply by ChrisTaylorDeveloper** [reply]
I am using the Gnome terminal.  I have tried setting my terminal colors to Gnome Light, Gnome Dark and Gnome Solarized and the problem still exists.  I have also tried setting the Gnome terminal to use the systme colors i.e. Ubuntu purple and the problem still exists.  I would be happy to install the GitHub colors into my terminal, however when I go to https://github.com/projekt0n/github-theme-contrib I don't see an option for Gnome?
